### PR TITLE
Cleaning up the 7.1 branch

### PR DESCRIPTION
**Pull request by vasanthrajams**
Signed-off-by: vasanthrajams <vasanthrajams@microsoft.com>

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.1/keys.json [hunk 1]
```diff
@@ -988,9 +988,7 @@
             "EC",
             "EC-HSM",
             "RSA",
-            "RSA-HSM",
-            "oct",
```
Comment (heaths) [inline specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.1/keys.json]: "oct" was actually in 7.0 so you can't remove it. That would be a breaking change. Presumably, the Key Vault service already responds with an error if you try to use it.

**Review by heaths — APPROVED** [review]
Be sure to add "oct" back everywhere, but otherwise LGTM.

**Review by heaths — APPROVED** [review]

**Comment by heaths** [comment]
@tjprescott I verified that all ModelValidation errors are expected, given the validation bug. Can you merge this?

**Comment by heaths** [comment]
Oh, one more thing is needed. Please delete the files that rbac.json referenced, as shown in the Avocado validation errors: https://dev.azure.com/azure-sdk/public/_build/results?buildId=261331&view=logs&jobId=f10f5372-40ed-54fc-caa4-b1cb5bd1252b&j=f10f5372-40ed-54fc-caa4-b1cb5bd1252b&t=d3b2b841-a93b-5fa5-47c3-d8c4f6eae6e3

**Comment by tjprescott** [comment]
@heaths it won't let me merge based on the failing checks.

**Comment by allenjzhang** [comment]
It is pushing to dev-branch as part of feature branch clean-up. Bypassing checks.

**Comment by heaths** [comment]
Also note that the checks that are failing are:

1. A bug in the Go generation (@jhendrixMSFT do you know anything about this?)
2. A bug in OpenAPI validation we can't work around or suppress.

@allenjzhang could you disable requiring a clean build on the branch policy we modified? I will be sure (and have been) the only failures are what's listed above. For example, I take the ModelValidation errors and use a regex search to make sure that all failures are the `MISSING_REQUIRED_PARAMETER` failure that is the bug in OpenAPI validation.

**Comment by jhendrixMSFT** [comment]
This is not a bug in the generator, it's due to invalid configuration in readme.go.md
```
System.InvalidOperationException: codegen for preview swagger Microsoft.KeyVault/preview/7.1/certificates.json must be under a preview subdirectory
```
This will need to be fixed.

**Pull request merged**